tests/e2e/common-operator-integ-suite.sh

```diff
@@ -191,18 +220,50 @@ if [ "${SKIP_BUILD}" == "false" ]; then
     get_internal_registry
   fi
 
-  # BUILD AND PUSH IMAGE
-  build_and_push_image
+  build_and_push_operator_image
 
-  if [ "${OCP}" == "true" ]; then
-    # This is a workaround
-    # To avoid errors of certificates meanwhile we are pulling the operator image from the internal registry
-    # We need to set image $HUB to a fixed known value after the push
-    # This value always will be equal to the svc url of the internal registry
-    HUB="image-registry.openshift-image-registry.svc:5000/sail-operator"
+  # If OLM is enabled, deploy the operator using OLM
+  if [ "${OLM}" == "true" ] && [ "${SKIP_DEPLOY}" == "false" ]; then
+    # Set the platform to Kubernetes by default.
+    # We are skipping the deploy via OLM test on OCP because the workaround to avoid the certificate issue is not working.
+    # Jora ticket related to the limitation: https://issues.redhat.com/browse/OSSM-7993
+    HELM_PLATFORM="kubernetes"
```

Comment:
Here is the case I mentioned above, HELM_PLATFORM is ignored unless it's 'openshift'

Reply:
I added some comments to clarify why we are setting it to `Kubernetes` another option is to change the variable name to OCP and fill it empty in the call of the target to not set the helm flag. But I think is more explicit to set it to kubernetes